Support for VS2017.

mbadawi23 opened the pull request:
Upgraded solution to support VS2017 and VS2015. Unfortunately it is a one way upgrade, which means it will have to be build using VS2017 or MSBuild 15.0.

melkammar-w committed:
Upgraded solution to support VS2017 and VS2015. Unfortunatly it is a one way upgrade, which means it will have to be build using VS2017 or MSBuild 15.0.

guyboltonking commented:
Thanks!  I'd rather the extension still built in VS2015; is it possible to have separate .sln, .csproj and .vsixmanifests for VS2017?

melkammar-w committed:
Added solution and project files for VS2015 and VS2017.

mbadawi23 committed:
Create README.md

Added description, how to build and how to use sections.

mbadawi23 commented:
I did as you requested, I renamed the original solution and project files, and added new ones to use with VS2017. Also, added some README file so others could use the extension easily.

guyboltonking commented:
Thanks again!
